### DISCUSSION TITLE
Send texts on a schedule, disconnected from triggering events?

### DISCUSSION THREAD

**Discussion by antoni-g** [berrystephenw/OctoText#125] [Ideas]
Hey all, after coming across needing the use case myself, I see it has been requested in the past to add a feature to send a text/notification on a schedule regardless of events occuring (ie, every 10 mins whether or not the printer is printing). Unless I'm missing something, I don't see the possibility to do so in the current version of the plugin.

Is this something that aligns with consensus on features for the plugin? I'm happy to develop this and open a PR.

**Comment by berrystephenw** [comment]
It wouldn't be too difficult to modify the code to implement notifications on a set interval. I'm not sure why you would want to be notified if there was no printing going on though.

**Reply by antoni-g** [reply]
The use case seems mostly to be able to keep an eye on something like an enclosure with a heater or similar to make sure things are okay, since for example setting up an enclosure remotely and wanting to keep an eye on it isn't always nicely aligned with print times.

The other use is for potential expansion of certain capabilities of Octoprint with SLA features (that may or may not strictly adhere to Octoprint's concept of "print in progress"), but that's a loftier goal 


I am planning on opening a pull request soon with this feature. Are there any standards for functional testing?

**Reply by berrystephenw** [reply]
I don't have a formal testing strategy, but all new features are released on the rc branch so if you could work off of that one it would be best.
